internal/service/datasync/sweep.go

```diff
@@ -511,6 +516,63 @@ func sweepLocationHDFSs(region string) error {
 	return nil
 }
 
+func sweepLocationObjectStorages(region string) error {
```

Comment:
TODO: refactor sweeper to new format + reuse one sweeper logic for all locations